packages/main/src/ToolbarButton.ts

```diff
@@ -24,6 +24,7 @@ import { registerToolbarItem } from "./ToolbarRegistry.js";
  * <code>import "@ui5/webcomponents/dist/ToolbarButton";</code>
  *
  * @constructor
+ * @abstract
```

Comment:
`@abstract` in terms of JSDoc means abstract class (such as ToolbarItem). Do we also use it to mark our abstract tags (such as ui5-option, etc...) I have forgotten. If so, then it's ok

Reply:
As I remember we mark classes without template with @abstract tag (like Option)